helfomat-infrastructure-jpa/src/main/java/de/helfenkannjeder/helfomat/infrastructure/jpa/EventId.java

```diff
@@ -0,0 +1,53 @@
+package de.helfenkannjeder.helfomat.infrastructure.jpa;
+
+import org.springframework.data.annotation.AccessType;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * @author Valentin Zickner
+ */
+@Embeddable
+@AccessType(AccessType.Type.FIELD)
+public class EventId implements Serializable {
+
+    @Column(name = "event_id")
```

Comment:
Here you use underscore. In some other places, you use camel case.

Reply:
fixed